tox.ini

```diff
@@ -43,6 +43,7 @@ commands =
 usedevelop = true
 deps =
     zope.testrunner
+    plone.app.robotframework  # undeclared dependency in plone.app.event
```

Comment:
We should use `plone.app.contenttypes[test]` in `setup.py`.  That should pull in p.a.robotframework.

Reply:
Is this non-circular then?

Reply:
I am not sure what you mean.

Reply:
@mauritsvanrees good catch! 🕵🏾